src/devices/wholeBodyDynamics/app/CMakeLists.txt

```diff
@@ -82,6 +82,15 @@ install_wbd_robots_files(YARP_ROBOT_NAME iCubHeidelberg01
                          EXTERNAL_YRI_CONF launch-wholebodydynamics-icubheidelberg01.xml
                          ROBOT_WBD_CONF    wholebodydynamics-icubheidelberg01-robot.xml)
 
+#
+# iCubHeidelberg01 is one of a kind (a iCub with only legs and torso) and so it has specific configuration files
+#
+install_wbd_robots_files(YARP_ROBOT_NAME iCubGenova04
```

Comment:
I am a bit confused by the comment connected with function, and in particula that this call will going to overwrite the files written in https://github.com/robotology/codyco-modules/pull/281/files#diff-4c6dd013f6e7d072ff01fd02e3ec8a42R60 .

Reply:
The comment is indeed wrong since it is talking about the previous code in line 80. This was created do to the fact that only iCubGenova04 and iCubGenova02 have strain2 sensors so they are the only ones that by default could use this configuration and we are not calibrating iCubGenova02 anytime soon to use temperature coefficients.  Another example of bad copy-paste-edit.

Reply:
Changed comment